Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -42,3 +42,18 @@ public struct PlotLegend {
     public var textSize: Float = 10
     public init() {}
 }
+
+public protocol Annotation {
+    var text: String { get set }
```

Comment:
You can remove this from Annotation protocol. It won't make sense to have a text in an ArrowAnnotation which we might create in the future.